examples/vitals/tesla.proto

```diff
@@ -9,8 +9,277 @@ syntax = "proto3";
 
 package teslapower;
 
+import "google/protobuf/any.proto";
 import "google/protobuf/timestamp.proto";
 
+message ExternalAuth {
+    ExternalAuthType type = 1;
+}
+
+enum ExternalAuthType {
```

Comment:
Most of these aren't directly related to the /api/devices/vitals endpoint. I documented them here in case someone else finds them useful